dataland-backend/src/main/kotlin/db/migration/V3__DeleteInviteMetaInfoTable.kt

```diff
@@ -0,0 +1,15 @@
+package db.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+
+/**
+ * This migration script creates an index for a faster search for company names and identifiers
```

Comment:
This description seems to be wrong

Reply:
Fixed. (Sorry, the commit message of the fix-commit is old and confusing)

Reply:
Looks good now